File: src/main/java/entity/Itinerary.java
package entity;

import jakarta.persistence.*;
import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Entity
public class Itinerary {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer itineraryId;
    
    private Integer dayNo;
    private String dayDetail;
    
    @ManyToOne
    @JoinColumn(name = "package_id")
    private TourPackage tourPackage;
}

File: DotNet_Backend/SafarnamaApplication/SafarnamaApplication/Models/Itinerary.cs
using System;
using System.Collections.Generic;

namespace SafarnamaApplication.Models;

public partial class Itinerary
{
    public int ItineraryId { get; set; }

    public int? PackageId { get; set; }

    public int? DayNo { get; set; }

    public string? DayDetail { get; set; }

    public virtual Package? Package { get; set; }
}
